.github/dependabot.yml

```diff
@@ -5,6 +5,14 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "weekly"
+    ignore:
+      - dependency-name: "k8s.io/client-go"
```

Comment:
Use regex to skip dependency of kubernetes? like `k8s.io/*`

Reply:
done

Reply:
Any specific reason to skip this dependency? i dont see any replace section in go.mod if we dont have hard requirements why not update the dependency to keep the dependency up to date?

Reply:
sometimes, it requires manual code changes so that's why skipped it.

Reply:
Not always, right? i that case, can we take it as a notification, fix it, and close the PR opened by the bot?

Reply:
in that case, we need to close the pr.

Reply:
If we do close a PR from the bot because it requires additional code changes and if it takes some time, do we know if the bot will keep opening the PR again every week? Either way, maybe we should start for now without this config and add it later if we find it necessary.

Reply:
AFAIK if we close the PR, bot will not open a new PR for same version every week unless there is a new release available.

Reply:
should we close it?

Reply:
Yes IMO, we should close it.

Reply:
okay